sdk/storage/storage-blob/src/Clients.ts

```diff
@@ -3200,6 +3386,7 @@ export class BlockBlobClient extends BlobClient {
     }
     super(url, pipeline);
     this.blockBlobContext = new BlockBlob(this.storageClientContext);
+    this._blobContext = new StorageBlob(this.storageClientContext);
```

Comment:
nit: seems inconsistent that one has `_` and the other doesn't.

Reply:
That's a special case. BlobClient already has blobContext. BlockBlobClient cannot inherit because of API extractor issue. So BlockBlobClient initializes `_blobContext` to differentiate.